src/objdump.py

```diff
@@ -434,7 +428,13 @@ def fn(pat: Match[str]) -> str:
 def objdump(
     o_filename: str, arch: ArchSettings, *, stack_differences: bool = False
 ) -> List[Line]:
-    output = subprocess.check_output(arch.objdump + [o_filename])
+    executable = None
+    for cand in arch.executable:
+        if shutil.which(cand):
+            executable = cand
+            break
```

Comment:
Can you handle the case where no executable is found by printing some nice error? (Would also be nice to do this detection only at startup, but I guess there's no good place for it, so eh. Maybe we should add an `@lrucache` to the detection function though)

Reply:
I would love to do it, but im not that familiar with python, its the first time i heard about lrucache 0:

Reply:
It's super simple to use, it just marks a function as "cache the response value and return it if someone passes the same parameters again". Totally fine if you want to leave it out though.